tests/python/unittest/test_loss.py

```diff
@@ -207,7 +207,7 @@ def test_ctc_loss():
     mx.test_utils.assert_almost_equal(l.asnumpy(), np.array([18.82820702, 16.50581741]))
 
 
-@with_seed(1234)
+@with_seed()
```

Comment:
nit: please refer to https://github.com/apache/incubator-mxnet/pull/11888/files#diff-cb652780258e73a9cd08568f38929aa2R2509 for adding some tracking for this test.

Reply:
addressed this change as well